.eslintrc

```diff
@@ -4,7 +4,8 @@
     "es6": true,
     "browser": true,
     "jest": true,
-    "node": true
+    "node": true,
+  "extends": ["plugin:prettier/recommended"]
```

Comment:
You can add this prettier plugin to line 2.

Reply:
:thumbsup_all: